notebooks/book2/03/schools8.ipynb

```diff
@@ -0,0 +1,1684 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        %pip install -qq blackjax
</pre><p>This line will create an error in colab</p><p>Instead write, </p><p><code>%pip install -qq jaxopt blackjax</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/121/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>